build.gradle

```diff
@@ -106,7 +106,7 @@ jlink {
     imageDir = file(JLINK_DIR)
     imageZip = file(JLINK_DIR + ".zip")
     addOptions '--add-modules', MODULES_TO_ADD, '--strip-debug', '--compress', '2',
-            '--no-header-files', '--no-man-pages'
+            '--no-header-files', '--no-man-pages', '--ignore-signing-information'
```

Comment:
Is there a downside / danger to adding this?

If not, should we merge into 1.4.0 as well?

If so, we should flag this PR in #767 as another thing we'd need to eventually get into 1.4.0.

Reply:
I will definitely flag this in #767 to address before merging to 1.4.0

Reply:
I think we do need to. I created a draft PR for discussion here: https://github.com/BrightSpots/rcv/pull/770